### PR TITLE
docs: add example with an initial value set in the input component

### PR DESCRIPTION
**Pull request by amontalvof**
fix: #1831 

## Changes proposed in this PR:
-

- [ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/nexxtway/react-rainbow/blob/master/CONTRIBUTING.md#submitting-a-pull-request).

@nexxtway/react-rainbow

### CODE REVIEW

File: src/components/Input/readme.md [hunk 1]
```diff
@@ -17,6 +17,33 @@ const containerStyles = {
 />;
 ```
 
+##### input base with an initial value set
```
Comment (TahimiLeonBravo) [inline src/components/Input/readme.md:20]: I think we should start adding a useful and short little description for the new interactive examples

**Review by TahimiLeonBravo — CHANGES_REQUESTED** [review]

File: src/components/Input/readme.md [hunk 2]
```diff
@@ -17,6 +17,33 @@ const containerStyles = {
...
+const handleChange = (e) => {
+    setValue(e.target.value)
+}
+
+const containerStyles = {
+    maxWidth: 700,
+};
+
+<Input
+    id="input-component-1"
```
Comment (LeandroTorresSicilia) [inline src/components/Input/readme.md:37]: The id must be unique on a web page, the first example already use `id="input-component-1"`
Also keep in mind when adding new examples and use the ids that the integration tests depend on this ids and on the example order too

**Review by TahimiLeonBravo — APPROVED** [review]

**Comment by TahimiLeonBravo** [comment]
@amontalvof let's do the new examples in this way:
- Each example we should have a title and a description
- It should look like the design https://zpl.io/29LEldY

**Comment by TahimiLeonBravo** [comment]
@amontalvof we should use a smaller font-size for this example
show as it looks like now
Screen Shot 2020-09-28 at 12 25 54 AM

**Pull request merged**